docker-compose.yaml

```diff
@@ -0,0 +1,40 @@
+version: "3"
+services:
+  db_recipes:
+    restart: always
+    image: postgres:11-alpine
+    volumes:
+      - ./postgresql:/var/lib/postgresql/data
+    env_file:
+      - ./.env
+
+  web_recipes:
+    image: vabene1111/recipes
+    restart: always
+    env_file:
+      - ./.env
+    volumes:
+      - staticfiles:/opt/recipes/staticfiles
+      - nginx_config:/opt/recipes/nginx/conf.d
+      - ./mediafiles:/opt/recipes/mediafiles
+    depends_on:
+      - db_recipes
+    command: ["./wait-for-it.sh", "db:5432", "--", "python", "app.py"]
```

Comment:
I think you should change "db:5432" by the name of the database container and in the .sh file get the port using the variable set on the .env file.